src/components/screens/login/networkSelector/networkSelector.js

```diff
@@ -1,204 +1,205 @@
-      ButtonComponent={SecondaryButton}
-      align="right"
-    >
-      {
-          networkList.map((network, key) => {
-            if (network.name === networkKeys.customNode) {
-              return (
-                <span
-                  className={`${styles.networkSpan} address`}
-                  key={key}
-                  onClick={() => onSelectNetwork(networkKeys.customNode)}
-                >
-                  {network.label}
-                  <div className={styles.inputWrapper}>
-                    <Input
-                      autoComplete="off"
-                      onChange={onChangeInput}
-                      name="customNetwork"
-                      value={address}
-                      placeholder="e.g. https://mainnet-service.lisk.io or 192.168.0.1:4000"
-                      size="xs"
-                      className={`custom-network ${styles.input} ${isValid ? '' : styles.errorInput}`}
-                      onKeyDown={e => e.keyCode === keyCodes.enter && onConnectToCustomNode(e)}
-                      isLoading={isValidationLoading}
-                      status={connected ? 'ok' : 'error'}
-                      feedback={validationError}
-                    />
-                    {
-                      !isValid
-                        ? (
-                          <span className={styles.customNodeError}>
-                            {validationError}
-                          </span>
-                        )
-                        : null
-                    }
-                  </div>
-                  {
-                    isCustomSelected
-                      ? (
-                        <div>
-                          <PrimaryButton
-                            disabled={connected}
-                            /* istanbul ignore next */
-                            onClick={onConnectToCustomNode}
-                            className={`${styles.button} ${styles.backButton} connect-button`}
-                            size="xs"
-                          >
-                            {connected ? t('Connected') : t('Connect')}
-                          </PrimaryButton>
-                        </div>
-                      )
-                      : null
-                  }
-                </span>
-              );
-            }
-
-            return (
-              <span
-                onClick={() => {
-                  onSelectNetwork(network.name);
-                  validateCorrectNode(network.name);
-                  setState({ connected: false });
-                  childRef.current.toggleDropdown(false);
-                }}
-                key={key}
-              >
-                {network.label}
-              </span>
-            );
-          })
-        }
-    </DropdownButton>
-  );
-};
-
-NetworkSelector.displayName = 'NetworkSelector';
-export default NetworkSelector;
+import React, { useRef, useState } from 'react';
+

```

Comment:
same with this file as well, somehow the changes are applied to whole file, please make sure to preserve the commit history 🙏🏼